targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_LPC54114/device/TARGET_LPC54114_M4/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2021 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-include(${MBED_PATH}/targets/TARGET_NXP/scripts/mbed_set_post_build_nxp.cmake)
+include(${mbed-os_SOURCE_DIR}/targets/TARGET_NXP/scripts/mbed_set_post_build_nxp.cmake)
```

Comment:
Did we miss this one?

Reply:
yes, let me search again entgire tree if I've missed any other

Reply:
fixed